apps/next-evans/components/core/Seo.tsx

```diff
@@ -0,0 +1,37 @@
+import { t } from '@friends-library/locale';
+import Head from 'next/head';
+import React from 'react';
+import { LANG } from '@/lib/env';
+
+type Props = {
+  title: string;
+  description: string;
+  withoutEnding?: boolean;
+} & (
+  | { documentPage: false }
+  | { documentPage: true; documentSlug: string; authorSlug: string; edition: string }
```

Comment:
hmm... this feels wrong to me, because we're significantly complicating the type passed to this component, just so we can dynamically assemble the og image (as far as i can tell). 

that makes me think that what we really want here is a `ogImage?: string;` prop, which falls back to the feature image when not supplied. then, the doc pages can supply their own og image, instead of all this other data.

Reply:
Yeah that makes way more sense